docs/dg/dev/architecture/coding-conventions.md

```diff
@@ -0,0 +1,2165 @@
+
+The application layers are aggregations of layers (see [Modules and Application Layers](https://docs.spryker.com/docs/dg/dev/architecture/modules-and-application-layers.html))
+- **Zed**: is a **multi-layer composition** of the 4 layers: `Presentation Layer`, `Communication Layer`, `Business Layer`, `Persistence Layer`.
+- **Yves**: is a **single-layered** `Presentation Layer` merged with `Communication Layer`.
+- **Glue**: is a **single-layered** `Communication Layer`.
+- **Client**: is a **single-layered** `Communication Layer`.
+- **Service**: is a **single-layered**, overarching `Business Layer`.
+- **Shared**: is a layer-overarching, stateless, abstraction library.
+
+## Zed
+
+`Zed` serves as the backend application layer responsible for housing all business logic and persisting data.
+```
+[Organization]
+└── Zed
+    └── [Module]
+        ├── Presentation
+        │   ├── Layout
+        │   │   └── [layout-name].twig
+        │   └── [name-of-controller]
+        │       └── [name-of-action].twig
+        │    
+        ├── Communication
+        │   ├── Controller
+        │   │   ├── IndexController.php
+        │   │   ├── GatewayController.php
+        │   │   └── [Name]Controller.php
+        │   ├── [CustomDirectory]
+        │   │   ├── [ModelName]Interface.php
+        │   │   └── [ModelName].php        
+        │   ├── Exception
+        │   ├── Plugin
+        │   │   └── [ConsumerModule]
+        │   │       └── [PluginName]Plugin.php
+        │   ├── Form
+        │   │   ├── [Name]Form.php
+        │   │   └── DataProvider
+        │   │       └── [Name]FormDataProvider.php
+        │   ├── Table
+        │   │   └── [Name]Table.php                
+        │   ├── navigation.xml       
+        │   └── [Module]CommunicationFactory.php
+        │            
+        ├── Business
+        │   ├── [CustomDirectory]
+        │   │   ├── [ModelName]Interface.php
+        │   │   └── [ModelName].php        
+        │   ├── Exception
+        │   ├── Installer
+        │   │   ├── [Module]InstallerInterface.php
+        │   │   └── [Module]Installer.php        
+        │   ├── [Module]BusinessFactory.php
+        │   ├── [Module]Facade.php
+        │   └── [Module]FacadeInterface.php
+        │
+        ├── Persistence
+        │   ├── [Module]EntityManagerInterface.php
+        │   ├── [Module]EntityManager.php
+        │   ├── [Module]QueryContainer.php
+        │   ├── [Module]PersistenceFactory.php
+        │   ├── [Module]RepositoryInterface.php
+        │   ├── [Module]Repository.php
+        │   └── Propel
+        │       ├── Abstract[TableName].php
+        │       ├── AbstractSpy[TableName]Query.php
+        │       ├── Mapper
+        │       │   └── [Module|Entity]Mapper.php
+        │       └── Schema
+        │           └── [organization_name]_[domain_name].schema.xml
+        │
+        ├── Dependency
+        │   ├── Client
+        │   │   ├── [Module]To[Module]ClientBridge.php
```

Comment:
Bridge|Adapter

Reply:
although you are 100% right here, i don't want to advertise the Adapters, as they are rare cases. i want to avoid that people start to define Adapters by mistake.